src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -101,7 +126,7 @@ const AccordionPanel = forwardRef(
             </Box>
           )}
         </Button>
-        <Box border={theme.accordion.border}>
+        <Box border={contentBorder}>
```

Comment:
Unfortunately, after applying the following code as discussed (in order to replace the Box in line 129):
```
    const { border: contentBorder } = theme.accordion;
    const content = active ? (
      <Box border={contentBorder}> {children} </Box>
    ) : (
      undefined
    );
    ...
        {animate ? (
          <Collapsible open={active}>{content}</Collapsible>
        ) : (
          { wrapper: content }
        )}
```
I've encountered a backward compatibility issue due to the default bottom border this is always being rendered whether or not children, animate or active exist.
The previous behavior was enforcing the render of the Box due to the out of the box border it provides, so it is not possible not to display it. we may want to think on a more sophisticated way to work around the current behavior, we can apply it on a separate PR.